include/evmc/evmc.hpp

```diff
@@ -346,13 +346,36 @@ class result : private evmc_result
     /// @param _gas_left     The amount of gas left.
     /// @param _output_data  The pointer to the output.
     /// @param _output_size  The output size.
-    result(evmc_status_code _status_code,
-           int64_t _gas_left,
-           const uint8_t* _output_data,
-           size_t _output_size) noexcept
+    explicit result(evmc_status_code _status_code,
+                    int64_t _gas_left,
+                    const uint8_t* _output_data,
+                    size_t _output_size) noexcept
       : evmc_result{make_result(_status_code, _gas_left, _output_data, _output_size)}
     {}
 
+    /// Creates the result without output.
+    ///
+    /// @param _status_code  The status code.
+    /// @param _gas_left     The amount of gas left.
+    explicit result(evmc_status_code _status_code = EVMC_INTERNAL_ERROR,
+                    int64_t _gas_left = 0) noexcept
+      : evmc_result{make_result(_status_code, _gas_left, nullptr, 0)}
+    {}
+
+    /// Creates the result of contract creation.
+    ///
+    /// @param _status_code     The status code.
+    /// @param _gas_left        The amount of gas left.
+    /// @param _create_address  The address of the possibly created account. This can be valid
```

Comment:
How is the address valid in creation failure?

Reply:
Check now.